src/Collection.php

```diff
@@ -2,15 +2,23 @@
 
 namespace OpenFoodFacts;
 
-use Iterator;
-
-class Collection implements Iterator
+/**
+ * @phpstan-implements \Iterator<number, Document>
+ */
+class Collection implements \Iterator
 {
-    private $listDocuments  = null;
-    private $count          = null;
-    private $page           = null;
-    private $skip           = null;
-    private $pageSize       = null;
+    public const defaultPageSize = 24;
+
+    /** @var array<int, Document> */
+    private $listDocuments  = [];
+    /** @var int */
+    private $count          = 0;
+    /** @var int */
+    private $page           = 0;
+    /** @var int */
+    private $skip           = 0;
+    /** @var int */
+    private $pageSize       = 0;
```

Comment:
I would suggest this:
```suggestion
    /** @var array<int, Document> */
    private array $listDocuments  = [];
    private int $count          = 0;
    private int $page           = 0;
    private int $skip           = 0;
    private int $pageSize       = 0;
```

Reply:
The typed properties is not compatible with PHP 7.1
https://stitcher.io/blog/typed-properties-in-php-74

Reply:
Ah - the fate of old code :D - we then might want to up the requirements to 7.3/7.4 (https://www.php.net/supported-versions.php) and add a new semantic release.